libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -36,55 +61,59 @@ object MapboxMetricsReporter : MetricsReporter {
         accessToken: String,
         userAgent: String
     ) {
-        mapboxTelemetry = MapboxTelemetry(context, accessToken, userAgent)
```

Comment:
`context` is unused now.

Reply:
yes, but it's public API so we cannot change it